src/calculator/index.ts

```diff
@@ -68,14 +66,15 @@ export function parseOverrides(buf: Buffer): Promise<Overrides> {
         }
       })
       .on("data", (data) => {
-        if (data["coefficient"] !== "0" && data["coefficient"] !== "1") {
+        const coefficient = Number(data["coefficient"]);
+        if (!isFinite(coefficient)) {
```

Comment:
good catch!

Reply:
@vacekj  I am not sure, this seems to be doing the right thing, can you double check?

Here we want the coefficient to be a finite number. So any number but `Inifnity` and `NaN`. If the number conversion fails it will return NaN so it will throw an error.

Let me know if I am missing something.

Reply:
But actually I realised that we should be using `Number.isFinite` instead of `isFinite` since we already coerce the value before.

Reply:
@vacekj did you have a chance to check this?